plugins/com.google.cloud.tools.eclipse.appengine.newproject.test/build.properties

```diff
@@ -4,3 +4,4 @@ bin.includes = META-INF/,\
                .
 javacSource=1.7
 javacTarget=1.7
+additional.bundles = org.eclipse.emf.ecore
```

Comment:
Odd: I must have rebased this change before pushing up as I shouldn't have seen any errors with your change the `MANIFEST.MF`.